src/fluree/db/api/transact.cljc

```diff
@@ -19,20 +19,21 @@
           opts))
 
 (defn stage
-  [db txn opts]
+  [db txn {:keys [raw-txn] :as opts}]
```

Comment:
The raw-txn allows for us to store the initial incoming transaction without touching it at all. It's a bit awkward because we need to expand it in order to figure out which ledger to load, so we need to pass it down.